src/event.js

```diff
@@ -64,6 +64,61 @@ export const customEvents = popup => {
 			}
 		},
 
+		applyDragEvent = ( element ) => {
+			let initialY,
+				finalY,
+				containerBodyStyle,
+				initialHeight
+
+			const container = element.component.wikipediapreview,
+				containerHeader = element.querySelector( '.wikipediapreview-header' ),
+				containerBody = element.querySelector( '.wikipediapreview-body' ),
+				handleTouchStart = ( e ) => {
+					initialY = e.touches[ 0 ].clientY
+					containerBodyStyle = window.getComputedStyle( containerBody )
+					initialHeight = Number( containerBodyStyle.height.slice( 0, -2 ) )
+				},
+
+				handleTouchMove = ( e, isHeader ) => {
+					const clientY = e.touches[ 0 ].clientY,
+						offset = initialY - clientY,
+						currentHeight = initialHeight + offset,
+						expanded = element.querySelector( '.wikipediapreview.expanded' )
+
+					finalY = clientY
+					if ( !isHeader && !expanded || isHeader && expanded ) {
+						containerBody.style.maxHeight = currentHeight + 'px'
+					}
+				},
```

Comment:
insert the code `e.preventDefault()` here to prevent lagging sometimes

Reply:
Looks like adding `e.preventDefault()` messes with the text scrolling actually

Reply:
Opsss, then better don't have this code.